pkgs/top-level/all-packages.nix

```diff
@@ -8221,7 +8221,11 @@ let
   oniguruma = callPackage ../development/libraries/oniguruma { };
 
   openal = openalSoft;
-  openalSoft = callPackage ../development/libraries/openal-soft { };
+  openalSoft = callPackage ../development/libraries/openal-soft {
+    alsaSupport = if stdenv.isDarwin then false else true;
```

Comment:
`if foo then false else true` is just `!foo`. It also seems easier to change the default values in the openal-soft expression rather than to leave the defaults in place and always override them.